submissions/OlexiyDobroskok/OOP_exercise/index.js

```diff
@@ -0,0 +1,159 @@
+const inhabitantDefaultProperties = {
+  species: "No species",
+  name: "No name",
+  gender: "No gender",
+  saying: "Hello",
+  friends: "",
+};
+
+const humanDefaultProperties = {
+  hands: 2,
+  legs: 2,
+};
+
+const petsDefaultProperties = {
+  paws: 4,
+};
```

Comment:
It would be a better approach to have defaults within class definitions. Not necessarily in form of explicit const definitions. KISS

Reply:
done